hack/test-cmd.sh

```diff
@@ -51,7 +51,7 @@ function find_tests {
   cd "${OS_ROOT}"
   find "${1}" -name '*.sh' | sort -u
 }
-tests=( $(find_tests ${1:-test/cmd}) )
+tests=( 'test/cmd/admin.sh' ) #$(find_tests ${1:-test/cmd}) )
```

Comment:
It would be nice to be able to do this using arguments to `hack/test-cmd`. I'll work on that.